packages/lib/src/vis/heatmap/HeatmapVis.tsx

```diff
@@ -100,7 +103,9 @@ function HeatmapVis(props: Props) {
       >
         <Pan />
         <Zoom />
-        <SelectToZoom />
+        <XAxisZoom disabled={keepRatio} />
+        <YAxisZoom disabled={keepRatio} />
```

Comment:
Now adding axial zooming for Heatmap and RGB vis when there is no need to keep the ratio